apps/migrator-e2e/src/integration/migrator.spec.ts

```diff
@@ -138,4 +140,9 @@ describe('Migrator app', () => {
       migrations.forEach((migration: IMigration) => verifyMigration(migration))
     })
   })
+
+  it('Has no detectable a11y violations on load', () => {
+    // Test the page at initial load
+    cy.checkA11y()
```

Comment:
Are there ways to output what the failures are from here? I see in the logs that there are 2 failing items, but not what they are.

Reply:
The plugin outputs more detail on each caught issue in the console with links to a11y references. We can look into customizing what gets displayed in the command log.